modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -33,7 +33,33 @@ This {release} version of the software and programming language should not be us
 
 == Highlights of what's new in {release}
 
-The {release} release primarily includes user-visible new features, internal enhancements, and fixes and updates to the version of {proglang} and the {proglang} base library that are supported. 
+The {release} release primarily includes internal enhancements, and fixes and updates to {proglang}, the {proglang} base library, and Candid. 
+
+The most significant new features and fixes include the following minor updates to DFX, Candid, and Motoko.
+
+=== DFX
+
+* A fix to `+dfx upgrade+` that prevents a `+main+` thread panic when `+Option::unwrap()+` is called on a `+None+` value. 
+* A fix to the name of the language service command to restore its compatibility with IDEs. 
+
+=== Motoko
+
+* The {proglang} compiler supports specifying command-line arguments using `+--args <file>+` and `+--args0 <file>+`.
++
+The new options enable the compiler to read newline and NUL terminated arguments from a specified file name.
```

Comment:
```suggestion
```
This is not in this release, but the next release. The changelog above the version numbers means it's in a future release.